packages/frontend/src/contexts/User.tsx

```diff
@@ -1,6 +1,6 @@
 import { Identity } from '@semaphore-protocol/identity'
 import { DataProof } from '@unirep-app/circuits'
-import { UserState } from '@unirep/core'
+import SocialUserState from './Userstate'
```

Comment:
I think you can just change the import instead of changing the name
```ts
import UserState from './Userstate'
```
so the following type does not need to be changed

In the future if the UserState is updated in `@unirep/core` you can just change the import back